### PR TITLE
Docker task 2 Eftakhar

### PR DESCRIPTION
**Pull request by shoccho**
Fixes #

- [x] Does not depend on other PRs
- [ ] Depends on other PR being merged first: #

## What kind of change does this PR introduce (check at least one)

- [ ] Bugfix
- [ ] Improvement
- [x] New feature
- [ ] Refactor
- [ ] Infrastructure
- [ ] Other (please specify)

## How would you describe the changes in a changelog (1-2 lines)?
Inclueded dockercompose file and added two variants of docker file and docker compose
## What was done
Added docker-compose.yml for basic react deployment
Added Dockerfile.prod for new nginx deployment
Added docker-compose.prod for nginx deployment 
## How to test

- [ ] Needs regression testing (add QA needed label)

## Deploy risk & follow up

Please assess risk & follow up on deployment of this PR. Elaborate your choice if selecting anything other than 'No risk'.

- [x] No risk
- [ ] Low
- [ ] High

## Reminder list

- Determine whether REST API Documentation should be updated
- Make an assessment on whether unit tests should be updated and/or added
- Ensure follow up tasks are linked to this PR

QA - if regression testing was requested:

- [ ] QA has verified

### CODE REVIEW

File: frontend/react-client/Dockerfile.prod [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+# Defining a workir so our apllication code is in a specific folder
+# Organizing our container
+WORKDIR /app
+
+# Copying package.json as it includes project defination our our react project and is required to run the project
+#
+# Only copying this file specifily 
+#   because this file is the least likely to change
+#   so we can use cached version making our build faster
+COPY package.json .
```
Comment (musfique-m) [inline frontend/react-client/Dockerfile.prod:16]: Please also copy `package-lock.json`.

**Review by musfique-m — CHANGES_REQUESTED** [review]
Please take a look at the comment to copy package-lock.json.
Please also remove comments from Dockerfile.prod.

File: frontend/react-client/Dockerfile.prod (hunk 1 above)
Comment (shoccho) [reply frontend/react-client/Dockerfile.prod]: Resolved this in the commit [bfb8ccd](https://github.com/CraftsmenLtd/coe-batch-q322/pull/299/commits/bfb8ccd7dae9ae29b4b1c6cfb1d374bfa3cb0547)

**Review by musfique-m — APPROVED** [review]
Looks good.

**Pull request merged**